stock_orderpoint_automatic_creation/readme/USAGE.rst

```diff
@@ -0,0 +1,3 @@
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/153/11.0
```

Comment:
Do not put this image in USAGE.rst, it's now generated automatically as a readme badge.